### ISSUE TITLE
From 0.17.0 The custom scalars does'nt seem to work with interfaceType

### ISSUE THREAD

**Issue by nghiepit** [graphql-nexus/schema#622]
After upgrade to `0.17.0` or `0.18.0` my custom scalars does not work.

> [ERROR] 10:06:55 TypeError: t.datetime is not a function

My scalars
```ts
import {asNexusMethod} from '@nexus/schema';
import {DateTimeResolver} from 'graphql-scalars';

export const GQLDateTime = asNexusMethod(DateTimeResolver, 'datetime');
```

And schema

```ts
import * as schema from '@nexus/schema';

export const CreatedAt = schema.interfaceType({
  name: 'CreatedAt',
  definition(t) {
    // Does'nt works
    t.datetime('createdAt', {
      resolve(field) {
        return new Date(field.created_at) as any;
      },
    });
    t.resolveType(() => null);
  },
});

export const User = schema.objectType({
  name: 'User',
  definition(t) {
    t.implements('CreatedAt');
    t.id('id');
    t.datetime('last_login_at'); // It Works
  },
});

```

Reproduction https://github.com/nghiepit/nexus-scalar-bug
If downgrade to `0.16.0` it work again.
Maybe related https://github.com/graphql-nexus/schema/issues/565

**Comment by Mika83AC** [comment]
I can confirm this issue starting with 0.17.0. Also still present in 0.18.0.

**Comment by tgriesser** [comment]
Yep, just ran into this myself. Will look to get a fix in before 0.19
